src/smt/proof_post_processor.cpp

```diff
@@ -1015,6 +995,17 @@ Node ProofPostprocessCallback::expandMacros(ProofRule id,
   return Node::null();
 }
 
+Node ProofPostprocessCallback::addExpandStep(ProofRule id,
+                                             const std::vector<Node>& children,
+                                             const std::vector<Node>& args,
+                                             CDProof* cdp)
+{
+  // For now, this is a (locally) recursive call to expand macros; alternatively
+  // we could add the step to cdp and allow the proof node updater to call us
+  // again.
```

Comment:
I assume this is a placeholder for a future optimization? I think it'd be best to say so because it's kinda mystifying why to add this method for now.

Reply:
This optimization I found was not important, however it is nice to have a standard helper method for recursively calling the expandMacros.  I will comment on this.